### PR TITLE
Add check for broken Ddoc

### PR DESCRIPTION
**Pull request by wilzbach**
Follow-up automation to https://github.com/dlang/phobos/pull/5523

Can be enabled after https://github.com/dlang/phobos/pull/5526

### CODE REVIEW

**Comment by CyberShadow** [comment]
```
grep: Unmatched [ or [^
```

**Comment by quickfur** [comment]
Sigh. The shell is interpreting `$]` because you used double quotes rather than single quotes.

File: posix.mak [hunk 1]
```diff
@@ -639,6 +639,8 @@ test: $(ASSERT_WRITELN_BIN)_test all
 	@grep -n '[[:blank:]]$$' $$(find . -type f -name "*.dd") ; test $$? -eq 1
 	@echo "Searching for undefined macros"
 	@grep -n "UNDEFINED MACRO" $$(find $(DOC_OUTPUT_DIR) -type f -name "*.html" -not -path "$(DOC_OUTPUT_DIR)/phobos/*") ; test $$? -eq 1
+	@echo "Searching for undefined ddoc"
+	@grep -rn "[$$](" $(DOC_OUTPUT_DIR)/phobos-prerelease ; test $$? -eq 1
```
Comment (quickfur) [inline posix.mak]: Change `"[$]("` to `'[$]('` to fix it.

Comment (quickfur) [inline posix.mak]: I'm not sure using `$$` is the right solution. That may just be another magic variable...

**Comment by JackStouffer** [comment]
One would assume that this would be a feature in DMD.

**Comment by quickfur** [comment]
File an enhancement request?

**Comment by CyberShadow** [comment]
This needs to exclude .verbatim files, as those intentionally consist of DDoc macros.

**Comment by wilzbach** [comment]
@CyberShadow  - this is finally green & passing :)

**Pull request merged**